documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -190,55 +169,9 @@ ifdef::product[]
 * Setting up AMQ Streams storage
 * Installing {registry}
 
-. Extract the {prodname} connector archive(s) to create a directory structure for the connector plug-in(s). If you downloaded and extracted the archive for each {prodname} connector, the structure looks like this: 
```

Comment:
This seems not right to remove. Note the Debezium container image is only used for community, this is not used by the downstream product. Instead, the AMQ Streams image for Kafka Connect is used there, and procedure for that one still is the same as before at this point.

Reply:
That's poor. That's means a longer way for downstream product users then. I hope we can push the change downstream soon then. :crossed_fingers:

Reply:
Yes, we hope to improve this, but it entails some subtle complexities. Remind me, and I'll explain it in the next call.